lib/integrations/awsoidc/deployservice.go

```diff
@@ -80,13 +71,22 @@ const (
 	serviceStatusActive = "ACTIVE"
 	// serviceStatusDraining is the string representing an DRAINING ECS Service.
 	serviceStatusDraining = "DRAINING"
-)
 
-var (
+	// Ensure Cpu and Memory use one of the allowed combinations:
+	// https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_definition_parameters.html
+	taskCPU = "2048"
```

Comment:
Nit: Just to add clarification.
```suggestion
	taskCPU = "2048" // 2 vCPUs
```